app/views/automated_tests/_form.html.erb

```diff
@@ -7,7 +7,10 @@
   <script>
     document.addEventListener('DOMContentLoaded', () => {
       window.autotestManagerComponent = makeAutotestManager(document.getElementById('autotest-manager'),
-        { assignment_id: <%= @assignment.id %>});
+        { assignment_id: <%= @assignment.id %>,
+          grouping_id: <%= @assignment.groupings.joins(student_memberships: :user)
+                                      .where('users.type': 'TestStudent').first.id %>,
+          enableSubdirs: <%= allowed_to? :manage_subdirectories?, @assignment, with: SubmissionPolicy %> });
```

Comment:
not needed. this permission only applies to actual student submissions. We should never restrict this for the solutions file manager